powerauth-webflow/src/main/js/components/operationDetail.js

```diff
@@ -103,7 +103,7 @@ export default class OperationDetail extends React.Component {
                         {this.props.context.formData.parameters.map((item) => {
                             if (item.type === "AMOUNT") {
                                 return (
-                                    <div className="row attribute" key={item.label}>
+                                    <div className="row attribute" key={item.id}>
```

Comment:
Label and ID have completely different meaning. There must be no two items with the same ID. But there can be many items with the same label...

Reply:
Exactly, which is why we should use ID in this place, because react requires they key to be unique.

Reply:
Sorry... I see it now.